static/js/root.jsx

```diff
@@ -184,14 +190,21 @@ function Restaurants(props) {
             );
      }
 
-function RestaurantDetails() {
-    const [name, setName] = React.useState("")
-    const [website, setWebsite] = React.useState("")
-    const [vicinity, setVicinity] = React.useState("")
-    const [hours, setHours] = React.useState([])
-    const [googleRating, setGoogleRating] = React.useState(0)
+////////////////////////////////////////////////////////////////////////////////////////////////////
 
+function RestaurantDetails() {
```

Comment:
do not put all components into one file. 
usually the convention is one React Component per file, or at least one big component (>20 lines) per file.

Reply:
please move these components into their own files.